samples/wifi/shutdown/README.rst

```diff
@@ -28,6 +33,7 @@ The sample:
 #. Shuts down the Wi-Fi driver.
```

Comment:
Can we be more verbose here? Proposal:
"Brings down the Wi-Fi network interface, which automatically directs the Wi-Fi driver to power down the nRF70 device.